StortSpelProjekt/Engine/src/Misc/AssetLoader.cpp

```diff
@@ -19,20 +19,26 @@
 #include "../Renderer/Texture/Texture2DGUI.h"
 #include "../Renderer/Texture/TextureCubeMap.h"
 
-#include <DirectXMath.h>
+#include "EngineMath.h"
 
 AssetLoader::AssetLoader(ID3D12Device5* device, DescriptorHeap* descriptorHeap_CBV_UAV_SRV, const Window* window)
 {
 	m_pDevice = device;
 	m_pDescriptorHeap_CBV_UAV_SRV = descriptorHeap_CBV_UAV_SRV;
 	m_pWindow = const_cast<Window*>(window);
 
+	std::map<TEXTURE2D_TYPE, Texture*> matTextures;
 	// Load default textures
-	LoadTexture2D(m_FilePathDefaultTextures + L"default_albedo.dds");
-	LoadTexture2D(m_FilePathDefaultTextures + L"default_roughness.dds");
-	LoadTexture2D(m_FilePathDefaultTextures + L"default_metallic.dds");
-	LoadTexture2D(m_FilePathDefaultTextures + L"default_normal.dds");
-	LoadTexture2D(m_FilePathDefaultTextures + L"default_emissive.dds");
+	matTextures[TEXTURE2D_TYPE::ALBEDO]		= LoadTexture2D(m_FilePathDefaultTextures + L"default_albedo.dds");
+	matTextures[TEXTURE2D_TYPE::ROUGHNESS]	= LoadTexture2D(m_FilePathDefaultTextures + L"default_roughness.dds");
+	matTextures[TEXTURE2D_TYPE::METALLIC]	= LoadTexture2D(m_FilePathDefaultTextures + L"default_metallic.dds");
+	matTextures[TEXTURE2D_TYPE::NORMAL]		= LoadTexture2D(m_FilePathDefaultTextures + L"default_normal.dds");
+	matTextures[TEXTURE2D_TYPE::EMISSIVE]	= LoadTexture2D(m_FilePathDefaultTextures + L"default_emissive.dds");
```

Comment:
fix tabs, looks bad atm

Reply:
Just looks like this on github. Looks fine in the actual code